app/controllers/admin/subdomains_controller.rb

```diff
@@ -47,6 +47,19 @@ def invite_current_user_to_subdomain(subdomain)
     end
   end
 
+  def invite_current_super_user_to_subdomain(subdomain)
+    Apartment::Tenant.switch subdomain.name do
+      User.invite!(
+        email: current_user.email, 
+        can_manage_web: true,
```

Comment:
these permissions should be in a constant defined in the user model. It should be `CANONICAL_USER_PERMISSIONS` -- that way we just have to edit that constant when we add new permissions